examples/inverse_folding/score_log_likelihoods.py

```diff
@@ -110,6 +116,8 @@ def main():
 
     model, alphabet = esm.pretrained.esm_if1_gvp4_t16_142M_UR50()
     model = model.eval()
+    
+    parser.add_argument("--nogpu", action="store_true", help="Do not use GPU even if available")
```

Comment:
@naailkhan28 This needs to be moved up before the call to `args = parser.parse_args()`

Reply:
Thanks for spotting this! I've added a commit to fix this :)